### DISCUSSION TITLE
Add an option for default comment sort

### DISCUSSION THREAD

**Discussion by dondretrotman** [thunder-app/thunder#379] [Ideas]
Currently comments default to top sort. Could a setting be added to change the default?

**Comment by micahmo** [comment]
I believe this is already available.

| ![image](https://github.com/thunder-app/thunder/assets/7417301/e5a7e1fa-6689-41af-a7d6-9164406e80f4) |
| - |

If you don't see this setting, maybe try one of the prereleases.

**Comment by dondretrotman** [comment]
Thanks, it's there in the latest pre-release but not in the latest release.
